### DISCUSSION TITLE
Adoptive meshing on grain boundaries

### DISCUSSION THREAD

**Discussion by vivy234** [neperfepx/neper#796] [General]
Hi,
I want to add refined mesh on the grain boundaries of a 3d simulated microstructure in order to see the strain localization on the grain boundary. Any suggestions or command of adoptive meshing of .tess file would be greatly appreciated.
Thanks.

**Comment by rquey** [comment]
This isn't available. Neper tries to generate a mesh with the same element size everywhere.
